### PR TITLE
options: index nixpkgs

### PR DESCRIPTION
**Pull request by MarcelCoding**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -71,6 +71,15 @@
                   optionsPrefix = "programs.nixvim";
                   urlPrefix = "https://github.com/nix-community/nixvim/tree/main/";
                 }
+                /*{
```
Comment (SuperSandro2000) [inline flake.nix:74]: What's with this?

File: flake.nix [hunk 2]
```diff
@@ -71,6 +71,15 @@
...
+                  optionsJSON = (import "${nixpkgs}/doc/doc-support/options-doc.nix" { inherit (pkgs) nixosOptionsDoc; inherit (nixpkgs) lib; }).optionsJSON + /share/doc/nixos/options.json;
+                  urlPrefix = "https://github.com/NixOS/nixpkgs/tree/master/";
+                  name = "nixpkgs";
+                }*/
+                {
+                  optionsJSON = (import "${nixpkgs}/nixos/release.nix" { }).options + /share/doc/nixos/options.json;
+                  urlPrefix = "https://github.com/NixOS/nixpkgs/tree/master/";
+                }
               ];
```
Comment (SuperSandro2000) [inline flake.nix:83]: We should set name here

File: flake.nix (hunk 1 above)
Comment (MarcelCoding) [reply flake.nix:74]: This should be the nixpkgs options. https://search.nixos.org/options?channel=24.05&from=0&size=50&sort=relevance&type=packages&query=nixpkgs.*

I want to check if there are already included in the other one.

File: flake.nix (hunk 2 above)
Comment (MarcelCoding) [reply flake.nix:83]: The name currently does not exist because ixx is not merged yet.

File: flake.nix (hunk 1 above)
Comment (SuperSandro2000) [reply flake.nix:108]: Should be there 
image

**Pull request merged**